src/System.Console/src/System/ConsoleColor.cs

```diff
@@ -8,6 +8,8 @@ namespace System
 
     public enum ConsoleColor
     {
+        Unknown = -1,
```

Comment:
I commented on this in my PR description:
>I've explicitly not added the named value to the contract, as a) we want this to work with the currently shipping code on desktop, and b) the main use case is not via direct access by the user but rather a value returned from the getters.
You disagree?  .NET code targetting .NET 4.6, .NET 4.5, etc., would no longer be compatible, and we really don't expect people to be using Unknown directly.